app/styles/sidebar.styl

```diff
@@ -0,0 +1,56 @@
+.con-wrapper
+    @extend $sidebar
+    @extend $navigation
+
+.con-cat-discovery
+    background: embedurl("../assets/sprites/icon-compass.svg") em(24px) center no-repeat
+    &.active
+        background-image: embedurl("../assets/sprites/icon-compass-active.svg")
+.con-cat-categories
+    background: embedurl("../assets/sprites/icon-grid.svg") em(24px) center no-repeat
+    &.active
+        background-image: embedurl("../assets/sprites/icon-grid-active.svg")
+.con-cat-pinned
+    background: embedurl("../assets/sprites/icon-pin.svg") em(24px) center no-repeat
+    &.active
+        background-image: embedurl("../assets/sprites/icon-pin-active.svg")
+
+
+@media (max-width: (1023/basefont)rem)
+    .con-cat-discovery
+    .con-cat-categories
+    .con-cat-pinned
+        background-position center top
+
+    // need this due to no css-modules usage in this app
```

Comment:
In fact the aside width introduced by $app-2panes-sticky is overwritten by $navigation when we don't use css-modules.